File: Semestrul 4/MPP2/Client-Server_2Limbaje/Problema8SC_CSharp/Networking/dto/EventDTO.cs
namespace Networking.dto;

[Serializable]
public class EventDTO
{
    public string Name { get; set; }
    public int Distance { get; set; }
    public long AgeGroupId { get; set; }

    public EventDTO(string name, int distance, long ageGroupId)
    {
        Name = name;
        Distance = distance;
        AgeGroupId = ageGroupId;
    }

    public override string ToString()
    {
        return $"EventDTO[{Name} {Distance} {AgeGroupId}]";
    }
}

File: Semestrul 4/MPP2/Client-Server_2Limbaje/Problema8SC/Problema8SC/Problema8Networking/src/main/java/problema8/network/dto/EventDTO.java
package problema8.network.dto;

import java.io.Serializable;

public class EventDTO implements Serializable {
    private String Name;
    private Integer Distance;
    private Long AgeGroupId;

    public EventDTO(String name, Integer distance, Long ageGroupId) {
        this.Name = name;
        this.Distance = distance;
        this.AgeGroupId = ageGroupId;
    }

    public String getName() {
        return Name;
    }

    public Integer getDistance() {
        return Distance;
    }

    public Long getAgeGroupId() {
        return AgeGroupId;
    }

    @Override
    public String toString() {
        return "EventDTO[" + Name + " " + Distance + " " + AgeGroupId + "]";
    }
}
